src/test/regress/spec/isolation_blocking_move_multi_shard_commands_on_mx.spec

```diff
@@ -60,7 +61,8 @@ step "s1-begin"
 
 step "s1-move-placement"
 {
-    	SELECT master_move_shard_placement(get_shard_id_for_distribution_column, 'localhost', 57637, 'localhost', 57638, shard_transfer_mode:='block_writes') FROM selected_shard;
+  SET citus.defer_drop_after_shard_move to off;
```

Comment:
Why is this needed? I'd expect it not to be necessary with the `shardstate != 4` changes.